help_to_heat/frontdoor/views.py

```diff
@@ -557,20 +572,32 @@ def get_context(self, request, session_id, *args, **kwargs):
             {
                 "question": schemas.summary_map[question],
                 "answer": self.get_answer(session_data, question),
-                "change_url": reverse("frontdoor:change-page", kwargs=dict(session_id=session_id, page_name=page_name)),
+                "change_url": self.get_change_url(session_id, question, page_name),
             }
             for page_name, questions in schemas.household_pages.items()
             for question in questions
-            if question in session_data
-            if question in schemas.summary_map
+            if self.show_question(session_data, question)
         )
         return {"summary_lines": summary_lines}
 
+    def show_question(self, session_data, question):
```

Comment:
Not showing property type summary line for park homes, even if we've got answer for property type